### ISSUE TITLE
Black screen when restoring Defender

### ISSUE THREAD

**Issue by HattDroid** [ionuttbara/windows-defender-remover#101]
Fresh install of Windows 11 with the latest update.

Using the N option to remove Defender and security components, supposedly in a safe way.
Unfortunately, when trying to restore them with the 'R' option, which seem to be right one, I only get a black screen as attached, with no way to repair my defender application.

![Screenshot 2023-11-24 205245](https://github.com/ionuttbara/windows-defender-remover/assets/33228236/adc81715-2945-460c-866f-9a4e259f2953)

**Comment by ionuttbara** [comment]
What is your Windows 11 version? In Windows 11 23H2 or build 22631 removing with N it makes desktop to not open , because removing the folder ``` C:\\Program Data\\Microsoft\\Windows Defender\\ ``` breaks Windows App SDK with elements and it reverts to Windows 10 file explorer and Windows 10 Task Manager.

**Comment by HattDroid** [comment]
Sorry, I should have mentioned it, yes, it's Win11 23H2 on a fresh install.

The option I pressed is "(N) for disabling Defender and Security Components (safe)", it rebooted my machine.
Then I pressed "(R) for enabling Defender and Security Components (restore actions where N is pressed)", it rebooted again, but unfortunately, it didn't seem to have restored Defender correctly as this is when I was facing the black screen.
